src/store/ChartState.js

```diff
@@ -152,6 +171,60 @@ class ChartState {
             this.stxx.draw();
         }
     }
+
+    ClearChart() {
+        if (!this.clearChart) return;
+        // Remove comparsions
+        for (const field in this.stxx.chart.series) {
+            this.stxx.removeSeries(field);
+        }
+        // Remove indiactors
+        for (const id in this.stxx.layout.studies) {
+            const sd = this.stxx.layout.studies[id];
+            CIQ.Studies.removeStudy(this.stxx, sd);
+        }
+        this.stxx.clearDrawings();
+        this.mainStore.crosshair.setCrosshairState(0);
+
+        this.mainStore.chart.changeSymbol(this.stxx.chart.symbol, 0);
+        this.mainStore.chartType.setType('mountain');
```

Comment:
It'd be better to create a constants file for all default value in the project. It should be done probably on another card. Please if it's possible for you, add a TODO before the line to help us whenever we want to refactor the project.